core/trino-spi/src/main/java/module-info.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+module trino.spi {
+    requires transitive com.fasterxml.jackson.annotation;
+    requires transitive com.google.errorprone.annotations;
+    requires transitive io.opentelemetry.api;
+    requires transitive jakarta.annotation;
+    requires transitive java.logging;
```

Comment:
Why transitive?

Reply:
I assume you mean for this specific import.  If so, yes I think you are correct and this one should not be transitive.

Reply:
What about the others, such as error prone annotations, Jakarta annotations, etc